app/views/dashboard/_navigation.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= render TabsComponent.new do |component| %>
+  <%= component.tab(path: dashboard_path) do %>
+    <strong>Overview</strong>
```

Comment:
Why are these being styled bold here in this way? They should either be styled bold for all sub navigation across the site using CSS, or they shouldn't be bold.

Reply:
All the designs I have seen had them in bold. I agree that should be done in css though

Reply:
Oh, right, I just checked the design and I understand now.

In that case, I think this is using the wrong component - the wireframes for the admin flow have the same thing and it looks like this on the real site:

![image](https://user-images.githubusercontent.com/472830/115730783-80615000-a37e-11eb-94bc-f5d12eaabed0.png)

Reply:
@callumacrae from what Alaric said there's just one component that does this but it needs to support bold font if at the top and normal font if inside the page. The component this is based on is  https://hmcts-design-system.herokuapp.com/components/sub-navigation . What I have done is correct and if I can't see any benefit from styling this in css in this case.

Reply:
![image](https://user-images.githubusercontent.com/472830/116215820-ef082a00-a73f-11eb-866d-bad07ebc6a92.png)

For reference, this is what the page I just posted a screenshot of above looked like in the prototypes.

In our implementation currently, there's more than one component that does this! Per the HMCTS design system, that component is for _sub_ navigation, but we're dealing with primary navigation here (I don't know why it was renamed to Tabs when it was copied into this repo - it shouldn't have been).

If we're going to go with the design in this PR, we should still use the primary navigation component, but we should change it where it's already used, too. We 100% shouldn't have inconsistent navigation between pages!